core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -331,7 +396,7 @@ private void tryStart() {
             // Wait until the process within the container has become ready for use (e.g. listening on network, log message emitted, etc).
             waitUntilContainerStarted();
```

Comment:
Submitted https://github.com/testcontainers/testcontainers-java/pull/1782 to make it even faster 🚀